test/nstemplatetier/nstemplatetier.go

```diff
@@ -160,6 +160,40 @@ func Base1nsTier(t *testing.T, spec toolchainv1alpha1.NSTemplateTierSpec, option
 	return Tier(t, "base1ns", spec, options...)
 }
 
+var Base1nsRevision = map[string]string{
+	CurrentBase1nsTemplates.Namespaces[0].TemplateRef:        CurrentBase1nsTemplates.Namespaces[0].TemplateRef,
+	CurrentBase1nsTemplates.Namespaces[1].TemplateRef:        CurrentBase1nsTemplates.Namespaces[1].TemplateRef,
+	CurrentBase1nsTemplates.Namespaces[2].TemplateRef:        CurrentBase1nsTemplates.Namespaces[2].TemplateRef,
+	CurrentBase1nsTemplates.ClusterResources.TemplateRef:     CurrentBase1nsTemplates.ClusterResources.TemplateRef,
+	CurrentBase1nsTemplates.SpaceRoles["admin"].TemplateRef:  CurrentBase1nsTemplates.SpaceRoles["admin"].TemplateRef,
+	CurrentBase1nsTemplates.SpaceRoles["edit"].TemplateRef:   CurrentBase1nsTemplates.SpaceRoles["edit"].TemplateRef,
+	CurrentBase1nsTemplates.SpaceRoles["viewer"].TemplateRef: CurrentBase1nsTemplates.SpaceRoles["viewer"].TemplateRef,
```

Comment:
would be nice to have different values for the key and value to simulate the end-goal where the values will contain names of the TTR CRs

Reply:
Yes i agree, i shall update it in this PR once we finalise the logic/tests .

Reply:
updated this too PTAL